code/modules/mining/mine_items.dm

```diff
@@ -33,7 +33,6 @@
 	new /obj/item/device/mining_scanner(src)
 	new /obj/item/weapon/storage/bag/ore(src)
 	new /obj/item/device/flashlight/lantern(src)
-	new /obj/item/weapon/shovel(src)
```

Comment:
Ah, they just have two separate lockers for miners, one that is locked and another that isn't. Re-adding the shovel.